MergerService/src/run.cs

```diff
@@ -439,5 +435,27 @@ private bool Validate(IData target, TileBounds bounds)
 
             return hasSameTiles;
         }
+
+        public void UpdateRelativeProgress(MergeTask task, long currentAmount, long totalAmount, ITaskUtils taskUtils)
```

Comment:
Didn't we say this should be part of `Task`?

Reply:
i decide it more relevant as some external logic of run.cs scope